src/util/BaseLoginController.js

```diff
@@ -101,6 +101,13 @@ define(['okta', 'q'], function (Okta, Q) {
     postRenderAnimation: function() {
       // Event triggered after a page is rendered along with the classname to identify the page
       this.trigger('pageRendered', {page: this.className});
+
+      // If a stateToken exists, return it in the event context
+      var transaction = this.options.appState.get('transaction') && this.options.appState.get('transaction').data;
```

Comment:
could create a derived prop `stateToken` in AppState and will be just `this.options.appState.get('stateToken')`?